api/v1beta1/event.proto

```diff
@@ -9,9 +9,597 @@ import "google/protobuf/wrappers.proto";
 import "api/v1beta1/event_data.proto";
 import "api/v1beta1/threat.proto";
 
+enum Id {
```

Comment:
I think `Id` on its own may be confusing, maybe `EventId`?

Reply:
make sense, since `trace.Event` also uses `EventId`, tks